lib/sql.js

```diff
@@ -1508,3 +1508,251 @@ SQLConnector.prototype.getInsertedId = function(model, info) {
 SQLConnector.prototype.executeSQL = function(sql, params, options, callback) {
   throw new Error(g.f('{{executeSQL()}} must be implemented by the connector'));
 };
+
+// Refactored Discovery methods
+
+SQLConnector.prototype.querySchemas = function(options) {
+  return '';
```

Comment:
We can probably use `INFORMATION_SCHEMA` as the default implementation as it's an ANSI standard implemented by many databases. See https://en.wikipedia.org/wiki/Information_schema.